install

```diff
@@ -111,6 +113,9 @@ while true; do
         --address-sanitizer)
             build_address_sanitizer=true
             shift ;;
+        --freorg-bkwdcomp)
+            build_freorg_bkwdcomp=true
```

Comment:
If off by default you need the option to turn it off as I described in the email

Reply:
@TorreZuk I think you meant "If **on** by default..." but in any case I agree with Torre, it looks like its been coded to be on by default (see my comment on line 44 above) and the command-line option is also to turn it on...

Reply:
Updated flags in latest